.github/workflows/merge-check.yml

```diff
@@ -0,0 +1,29 @@
+name: Merge checks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    name: Build project
+    runs-on: ubuntu-latest
+    if: ${{ !contains(github.event.head_commit.message, 'ci skip') }}
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Cache Gradle Files
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches/
+            ~/.gradle/wrapper/
+          key: cache-gradle-files
```

Comment:
Shouldn't the gradle version be a part of the cache key? https://github.com/actions/cache/blob/main/examples.md#java---gradle

Reply:
This is what we get for copying from other projects 😅 
Fixed!